jenkins-builds/6.0.UP04-17.08-mc

```diff
@@ -5,6 +5,7 @@
  Boost-1.65.0-CrayGNU-17.08.eb
  Charm++-6.8.0-CrayGNU-17.08.eb                     --set-default-module --hidden
  Charm++-6.8.0-CrayIntel-17.08.eb                   --hidden
+ CMake-3.9.1.eb
```

Comment:
and 6.0.UP04-17.08-gpu ?

Reply:
=> https://github.com/eth-cscs/production/pull/370

Reply:
--set-default-module ?

Reply:
--set-default-module ?

Reply:
I think it makes sense to use the flag `--set-default-module` if you have more than one release installed. So I believe that we can use the flag as soon as we decide to build a new release.